web/template/admin/admin_tabs/edit-course.gohtml

```diff
@@ -25,9 +25,9 @@
                 </div>
 
                 <div class="bg-white dark:bg-secondary rounded shadow p-4 my-5">
-                    <h2 class="text-2 text-2xl font-semibold">Camera Presets</h2>
+                    <h2 class="text-2 text-2xl font-semibold">Source Settings</h2>
```

Comment:
This could be more intuitive for lecturers:

```suggestion
                    <h2 class="text-2 text-2xl font-semibold">Lecture Hall Settings</h2>
```